### PR TITLE
Added Reviewers dashboard

### PR DESCRIPTION
**Pull request by vnbang2003**
Added Reviewers dashboard

### CODE REVIEW

File: junction/urls.py [hunk 1]
```diff
@@ -27,7 +27,11 @@
 
     # Proposals related
     url(r'^(?P<conference_slug>[\w-]+)/proposals/', include('junction.proposals.urls')),
-    url(r'^(?P<conference_slug>[\w-]+)/dashboard/', 'junction.proposals.views.dashboard', name='proposal-dashboard'),
+    url(r'^(?P<conference_slug>[\w-]+)/dashboard/reviewers/',
+        'junction.proposals.dashboard.reviewer_dahboard',
```
Comment (kracekumar) [inline junction/urls.py]: Can you fix the spelling mistake ?

File: junction/proposals/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            private_comment_count = ProposalComment.objects.filter(
+                proposal=proposal, deleted=False, private=True).count()
+            if private_comment_count:
+                by_reviewer[key_id][1] = by_reviewer[key_id][1] + 1
+            else:
+                by_reviewer[key_id][2] = by_reviewer[key_id][2] + 1
+
+    audience_dict = {
+        1: 'Beginner',
+        2:  'Intermediate',
```
Comment (kracekumar) [inline junction/proposals/dashboard.py]: Maintain consistency in spaces ?

File: junction/proposals/dashboard.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        'group_by_reviewer_section': by_reviewer,
+        'by_target_audience': by_audience
+    }
+
+    return render(request, 'proposals/dashboard.html', ctx)
+
+
+@login_required
+@require_http_methods(['GET'])
+def reviewer_dahboard(request, conference_slug):
```
Comment (kracekumar) [inline junction/proposals/dashboard.py]: Can you rename the function ?

**Comment by vnbang2003** [comment]
@kracekumar have update as per your suggestions.

**Pull request merged**